package.json

```diff
@@ -77,7 +77,7 @@
     "start-heroku": "npm run terra -- express-server --port $PORT --site './build' ",
     "terra": "node ./packages/terra-cli/bin/terra",
     "test": "npm run lint && npm run jest && npm run mocha && npm run wdio",
-    "wdio-site": "SITE=build npm run terra -- wdio --browsers chrome firefox",
+    "wdio-site": "npm run terra -- wdio --site build --browsers chrome firefox --locales en fr --formFactors small large",
```

Comment:
Was the locale and form factor scoping for testing?

Reply:
This PR should fix the issue on #572 where only the first locale/theme/formFactor/etc is used if multiple are passed down. This is for making sure that the subsequent values in the options are used as expected. Do you think adding this is unnecessary and creates too many screenshots?

Reply:
nah, it's fine